test1.js

```diff
@@ -3,3 +3,10 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+
+var baaaar = "";
+
+if(baaaar === "bar") {
+  console.log( 'heeeeey' )
```

Comment:
Codacy has a fix for the issue: [Replace `·'heeeeey'·)` with `"heeeeey");`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5800)
```suggestion
  console.log("heeeeey");
```